### ISSUE TITLE
A simple text edit is not working when used in flask framework

### ISSUE THREAD

**Issue by PrasannaInamdar** [vitalets/angular-xeditable#667]
I am creating a simple web app in flask python, where I want my data to be editable when clicked on it. I tried using a simple inline text edit feature of angular-xeditable. but it is not working with flask. when I download and test a demo starter page it works properly. My angular code stops working when I include 'xeditable' dependency. It does not even show "Howdy!!", it shows //myname//.

This is my code.

`<html>
<head>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.min.js"></script>
   <script src="http://code.jquery.com/jquery-2.0.3.min.js"></script> 
   <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="static/css/tableCss.css">
    <link href="angular-xeditable/css/xeditable.css" rel="stylesheet">
    <script src="angular-xeditable/js/xeditable.js"></script>
    
</head>
<body >

    

        <div ng-app="app" ng-controller="myctrl">
         <a href="#" editable-text="myname">// myname //</a>
         <table>
            <caption> Todo List</caption>
                <thead>
                    <tr>
                        <td>
                            <input type="text" name="text" ng-model="addMe">
                            <input type="submit"  name="my-form" value="Add Task" >
                        
                        </td><td>
                        Search :<input type="text"> 

                        </td>
                    </tr>
                    <tr>
                        <th>Task Id</th>
                        <th>Task</th>
                    </tr>
                </thead>
                
                <tbody>
                    
                    {% for user in User %}
                    <tr>
                        <td> {{ user.uid }} </td>
                        <td> {{ user.taskname }} 
                        
                            <span style="padding-right:3px; padding-top: 3px; display:inline-block;" ></span>
                            <input type="image" class="deleteImg" src="static/img/trash_can.png" height="15px" width="18px" name="removeId" value={{user.uid}} ></input> 
                             <input type="image" src="static/img/editbtn.svg" height="15px" width="18px" name="editId" value={{user.uid}} ></input> 
                            
                        </td>
                    </tr>
                    {% endfor %}
                </tbody>
            </table>

        </div>
   
    <script>
    var app = angular.module('app', ['xeditable']);
    app.controller('myctrl', ['$scope', function($scope) {
      $scope.myname="Howdy!!"
    }]);
    
    app.config(function($interpolateProvider) {
        $interpolateProvider.startSymbol('//').endSymbol('//');
    });
    
    
    </script>  
  
</body>
</html>
  `

**Comment by ckosloski** [comment]
Did you verify that xeditable.js is loading?
Any errors in your javascript console?

**Comment by PrasannaInamdar** [comment]
@ckosloski You are right, JS file is loaded now and it is working correctly.
